src/main/java/net/neoforged/neoforge/client/extensions/IBlockModelGeneratorsExtension.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.extensions;
+
+import net.minecraft.client.data.models.BlockModelGenerators;
+import net.minecraft.client.data.models.blockstates.MultiVariantGenerator;
+import net.minecraft.client.data.models.model.ModelLocationUtils;
+import net.minecraft.client.data.models.model.ModelTemplates;
+import net.minecraft.client.data.models.model.TextureMapping;
+import net.minecraft.data.BlockFamily;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.properties.BlockStateProperties;
+
+public interface IBlockModelGeneratorsExtension {
```

Comment:
This extension is not strictly necessary.
It is possible to create a custom BlockFamily that is incomplete and use it to perform these generations

Reply:
Please remove it.